py_ecc/optimized_bls12_381/constants.py

```diff
@@ -0,0 +1,50 @@
+from py_ecc.fields import (
+    optimized_bls12_381_FQ2 as FQ2,
+)
+
+# Ciphersuite BLS12381G2-SHA256-SSWU-RO paramters
+ISO_3_A = FQ2([0, 240])
+ISO_3_B = FQ2([1012, 1012])
+ISO_3_Z = FQ2([1, 1])
+P_MINUS_9_DIV_16 = 1001205140483106588246484290269935788605945006208159541241399033561623546780709821462541004956387089373434649096260670658193992783731681621012512651314777238193313314641988297376025498093520728838658813979860931248214124593092835  # noqa: E501
+SQRT_I = 1028732146235106349975324479215795277384839936929757896155643118032610843298655225875571310552543014690878354869257  # noqa: E501
+EV1 = 426061185569912361983521454249761337083267257081408520893788542915383290290183480196466860748572708974347122096641  # noqa: E501
+EV2 = 1288825690270127928862280779549770369920038885627059587892741294824265852728860506840371064237610802480748737721626  # noqa: E501
+
+positive_eigth_roots_of_unity = (
+    FQ2([1, 0]), FQ2([0, 1]), FQ2([SQRT_I, SQRT_I]), FQ2([SQRT_I, -SQRT_I]),
+)
+eighth_roots_of_unity = (
```

Comment:
conforming to the all-caps casing would be nice to maintain consistency.

Reply:
Done :)